packages/mask/src/tsconfig.json

```diff
@@ -31,6 +31,13 @@
     { "path": "../../plugins/Wallet" },
     { "path": "../../plugins/DAO" },
     { "path": "../../plugins/FileService" },
+    { "path": "../../plugins/RSS3" },
+    { "path": "../../plugins/Web3Profile" },
+    { "path": "../../plugins/example" },
+    { "path": "../../plugins/Debugger" },
+    { "path": "../../plugins/CyberConnect" },
+    { "path": "../../plugins/CrossChainBridge" },
+    { "path": "../../plugins/GoPlusSecurity" },
```

Comment:
I don't know why you rely on so many plugins which I think is unnecessary. Remove them if possible, it will speedup the type check.